Extension/src/common.ts

```diff
@@ -176,19 +176,19 @@ export function isUri(input: any): input is vscode.Uri {
 }
 
 export function isString(input: any): input is string {
-    return typeof(input) === "string";
+    return input && typeof(input) === "string";
```

Comment:
revert the changes in these helper functions and in `resolveVariables`. this is a result of not having the latest changes from master when you copied over your branch.